pandas/io/excel.py

```diff
@@ -68,40 +67,43 @@
     * 1 -> 2nd sheet as a DataFrame
     * "Sheet1" -> 1st sheet as a DataFrame
     * [0,1,"Sheet5"] -> 1st, 2nd & 5th sheet as a dictionary of DataFrames
-    * None -> All sheets as a dictionary of DataFrames
-
-sheetname : string, int, mixed list of strings/ints, or None, default 0
+    * None -> All sheets as a dictionary of DataFrames.
+sheetname : str, int, mixed list of strs/ints, or None, default 0
+    Name or index of sheet or sheets to read into a DataFrame or
+    dictionary of DataFrames.
```

Comment:
Just say "Alias of `sheet_name`." (sheet_name in backticks, they are rendered in the comment)